src/chef-server-ctl/plugins/filtered-dump.rb

```diff
@@ -1,6 +1,6 @@
 
 
-add_command_under_category "filtered-dump", "Debug Tools", "Generate a filtered dump of indexable Chef Objects for all organizations. Top-level data is captured; only object name is captured from object json" do
+add_command_under_category "filtered-dump", "Debug Tools", "Generate a filtered dump of indexable #{ChefUtils::Dist::Server::PRODUCT} Objects for all organizations. Top-level data is captured; only object name is captured from object json" do
```

Comment:
Since these are not Chef Infra Server only objects per se - It might be ok to use the replacement for Chef instead of 'Chef Infra Server'

Reply:
That makes sense, thank you! Fixing this up now.